_members/cameron-mattson.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Cameron Mattson
+image: images/team/cameron_mattson.jpg
+description: Data Scientist
+role: BioData Scientist
+aliases:
+  - Cameron Mattson
+  - Cameron W. Mattson
+  - C. Mattson
+  - cmattson
+  - mcameron
+links:
+  email: cameron.mattson@cuanschutz.edu
+  github: https://github.com/MattsonCam
+---
+
+Cameron graduated with his B.S. in Bioengineering at the University of Colorado, where he implemented and developed physiological models. In his capstone project, he collaborated with a cardiovascular surgeon to design a medical device allowing access to the axilla during first rip resection surgeries. Cameron is currently pursuing his Graduate degree in Computer Science with a focus in Data Science and Data Engineering from the University of Colorado Boulder. He has worked on a variety of machine learning and data science project with both large and small companies.
```

Comment:
Please move to a [1 sentence per line format](https://sive.rs/1s) for markdown.

I think I also see a typo rip --> rib?

Reply:
Good catch!